pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java

```diff
@@ -94,6 +94,10 @@
 
     private boolean autoUpdatePartitions = true;
 
+    private long backoffIntervalNanos = TimeUnit.MILLISECONDS.toNanos(100);
```

Comment:
This should only be defined in the `ClientBuilder` not for producer/consumers

Reply:
Well, I put the configuration there because I thought that was the only place where it would belong. Where else would the configuration go then? From what I saw, ```ProducerConfigurationData``` and ```ConsumerConfigurationData``` is the only way to define the configurations of a class.

Reply:
@ConcurrencyPractitioner I think what @merlimat suggested is `backoff` is the settings for a pulsar client, it is a common setting shared across the producers and consumers on one client. it would be good to add the settings to ClientBuilder and ClientConfigurationData. You can access the configuration in client object. Hope this clarify the comment.